build.sbt

```diff
@@ -0,0 +1,51 @@
+import PlayCrossCompilation._
+import uk.gov.hmrc.DefaultBuildSettings.defaultSettings
+
+enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning, SbtArtifactory)
+
+name := "work-item-repo"
+
+makePublicallyAvailableOnBintray := true
+
+majorVersion                     := 6
+
+defaultSettings()
+
+scalaVersion := "2.11.12"
+
+crossScalaVersions  := Seq("2.11.12", "2.12.6")
+
+libraryDependencies ++= LibDependencies()
+
+crossScalaVersions := Seq("2.11.8")
+
+resolvers := Seq(
+  Resolver.bintrayRepo("hmrc", "releases"),
+  "typesafe-releases" at "http://repo.typesafe.com/typesafe/releases/"
+)
+
+collaborators
```

Comment:
Maybe we could remove collaborators (don't quite see a reason for it) and we could make this file look like all other `build.sbt` i.e. val root = Project(name, ".").settings(...) rather then having all settings at root level. Just to keep it consistent with other services/libs